README.md

```diff
@@ -23,19 +25,22 @@
 ## Your bot list should:
 
 1. not display all bots on the homepage.
-2. not be hosted on Glitch.
-3. have an API with good documentation.
+2. not be hosted on any free hosting provider (Glitch, RedHat OpenShift, Heroku, etc.).
```

Comment:
Remove the examples as we once agreed to not give those as examples since people would complain "But what about xyz?"

Reply:
I added an etc. for a reason, and we want people to get an idea of what isn't allowed.

Reply:
We've had this same discussion before in previous iterations of this file, I am strongly against listing specific services.

Reply:
Works with me.